context/brand-context/springernature/scss/60-utilities/buttons.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * Button utilities
+ * These are for styled branded buttons
+ */
+
+// Default button
+
+.u-button {
+	border: $context--button-border-width-style transparent;
+	box-shadow: 0 0 0 1px $context--button-brand-color;
+
+	&:focus {
+		box-shadow: none;
+		outline: none;
+	}
+}
+
+// Button themes
+
+.u-button--primary {
+	@include button-primary
+}
+
+.u-button--secondary {
+	@include button-secondary;
+
+	&:hover {
+		box-shadow: none;
+		border: $context--button-border-width-style $context--button-brand-color;
+	}
+}
```

Comment:
we already have `default/scss/60-utilities/buttons.scss`. I think we need to find a way to incorporate these extra styles into the settings so that you only need to include the utility once (from default). We want to keep utilities as much as possible in the `default` brand, and we don't want to be requiring you include two files, one from each brand as there is a high possibility of user error.